src/pages/PatientDashboard/PatientDashboard.tsx

```diff
@@ -1,7 +1,7 @@
-import Title from './Title/Title.tsx'
-import CommentList from './CommentList/CommentList.tsx'
-import Graph from '../../components/graph/graph'
+import Graph from '../../components/Graph/graph'
```

Comment:
Could the graph file be started with Capital for consistency

Reply:
Fixed as well 👍